app/views/environments/_form.html.erb

```diff
@@ -1,28 +1,20 @@
 <%= form_for @environment, :url => (@environment.new_record? ? environments_path : environment_path(:id => @environment)) do |f| %>
   <%= base_errors_for @environment %>
-  <% if show_taxonomy_tabs? %>
-    <ul class="nav nav-tabs" data-tabs="tabs">
-      <li class="active"><a href="#primary" data-toggle="tab"><%= _('Primary') %></a></li>
-      <% if show_location_tab? %>
-        <li><a href="#locations" data-toggle="tab"><%= _('Locations') %></a></li>
-      <% end %>
-      <% if show_organization_tab? %>
-        <li><a href="#organizations" data-toggle="tab"><%= _('Organizations') %></a></li>
-      <% end %>
-    </ul>
+  <ul class="nav nav-tabs" data-tabs="tabs">
+    <li class="active"><a href="#primary" data-toggle="tab"><%= _('Primary') %></a></li>
```

Comment:
I'd suggest changing the tab text to "Environment" while you're here, since it will now be visible by default and "Primary" isn't descriptive. The other two forms have better text already.